heroic-core/src/main/java/com/spotify/heroic/http/write/WriteResource.java

```diff
@@ -52,7 +60,13 @@ public void metrics(
         @Suspended final AsyncResponse response, @QueryParam("group") String group,
         WriteMetricRequest write
     ) throws Exception {
-        httpAsync.bind(response, ingestion.useGroup(group).write(write.toIngestionRequest()),
-            r -> r);
+        final IngestionGroup ingestionGroup = ingestion.useGroup(group);
```

Comment:
Unrelated change to this commit?

Reply:
This happens because `EmptyMetricCollection` was removed in favor of `Optional<MetricCollection>`

Reply:
Ok 👍